tests/baselines/reference/jsFileFunctionOverloads.symbols

```diff
@@ -0,0 +1,87 @@
+=== tests/cases/compiler/jsFileFunctionOverloads.js ===
+/**
+ * @overload
+ * @param {number} x

```

Comment:
I'm not sure what happens if you find-all-ref or rename the overload parameters. In Typescript, they are not connected to the implementation signatures and don't rename.

Can you add two tests in cases/fourslash, one that renames an overload param and one that find-all-refs it, and shows that overload params are not connected to the implementation? There should be existing tests for `@param` that you can start from.

Reply:
I only tried this in VSCode, and it seems like your assumptions are correct, i.e. the overload parameters are generally ignored on both `find-all-ref` and `rename` operations.

I would be happy to add those tests, but can you please point me to an example where a similar test was already implemented? I am fairly new to this codebase and navigating around (especially within tests) is still challenging 😄

Reply:
I've added tests for `find-all-ref` and `rename` as requested.